src/get-started/localisation/index.md.njk

```diff
@@ -0,0 +1,101 @@
+---
+layout: layout-pane.njk
+title: Using localisation with GOV.UK Frontend
+section: Get started
+theme: How to guides
+order: 5
+description: Localisation allows you to translate GOV.UK Frontend components into other languages
+---
+
+Localisation allows you to translate GOV.UK Frontend components into other languages
+
+## Set up localisation using Nunjucks
+
+If you’re using the Nunjucks macros provided by GOV.UK Frontend, you can localise the UI text that appears within components by passing translation strings as named parameters.
+
+The individual components using these parameters include:
+
+* accordion
+* character count
+* error message
+* footer
+* header
+* summary list
+
+## Set up localisation with Javascript
+
+You can only pass translation strings to JavaScript with a list of key-value pairs.
+
+When using a list of key-value pairs, the keys must be in quote marks, for example, “YOUR_KEY”.
```

Comment:
Should this be talking about values being in quotes, rather than keys? The code examples below seem to contradict what we're saying here (keys not quoted, and keys using single quote)

Reply:
Having keys in quotation marks used to be necessary when we namespaced them with a dot, like `"accordion.show_all_section"`. This can be removed or repurposed to refer to keys now, though.

Reply:
```suggestion

```